components/Snackbar/snackbar.js

```diff
@@ -0,0 +1,21 @@
+const snackbarBaseline = document.querySelector(".btn-action-baseline");
+const snackbarLeading = document.querySelector(".btn-action-leading");
+
+snackbarBaseline.addEventListener("click", () => {
+  document.querySelector(".snackbar-baseline").classList.remove("hide");
+  document.querySelector(".snackbar-leading").classList.add("hide");
+  closeSnackbar();
+});
+
+snackbarLeading.addEventListener("click", () => {
+  document.querySelector(".snackbar-baseline").classList.add("hide");
+  document.querySelector(".snackbar-leading").classList.remove("hide");
+  closeSnackbar();
+});
+
+const closeSnackbar = () => {
+  setTimeout(() => {
+    document.querySelector(".snackbar-baseline").classList.add("hide");
+    document.querySelector(".snackbar-leading").classList.add("hide");
+  }, 3000);
```

Comment:
this timeout function will put two dom elements two hide at the same time, you can further work on it for only one at a time. it can be done by passing a parameter to the closeSnackbar function.

Reply:
Done. Thanks for the great suggestions.